### ISSUE TITLE
Can't use plugin

### ISSUE THREAD

**Issue by keyser75000** [pluginsGLPI/openvas#9]
Hi,

I try to use this plugin but nothing seems to work.
My openvas server is working and reachable by GLPI (test KO from plugin configuration, telnet OK on specified ports).
In Tools > Openvas > Vulnerability, I see nothing.
In Tools > Openvas > Tasks, I can't create any tasks.
All dropdonws in these menus are empty.

=> in php-errors.log I see :
`2017-02-23 17:07:45 [19@xxx]` 
`  *** PHP Notice(8): Trying to get property of non-object`
`  Backtrace :`
`  plugins/openvas/inc/omp.class.php:558`
`  plugins/openvas/inc/task.class.php:170             PluginOpenvasOmp::displayDropdown()`
`  plugins/openvas/front/task.form.php:58             PluginOpenvasTask::showAddTaskForm()`
`2017-02-23 17:07:45 [19@xxx]`
`  *** PHP Warning(2): Invalid argument supplied for foreach()`
`  Backtrace :`
`  plugins/openvas/inc/omp.class.php:558`
`  plugins/openvas/inc/task.class.php:170             PluginOpenvasOmp::displayDropdown()`
`  plugins/openvas/front/task.form.php:58             PluginOpenvasTask::showAddTaskForm()`
`2017-02-23 17:07:47 [19@xxx]`
`  *** PHP Notice(8): Undefined index: scanner`
`  Backtrace :`
`  plugins/openvas/inc/omp.class.php:578`
`  plugins/openvas/front/task.form.php:40             PluginOpenvasOmp::addTask()`
`2017-02-23 17:07:47 [19@xxx]`
`  *** PHP Notice(8): Undefined index: config`
`  Backtrace :`
`  plugins/openvas/inc/omp.class.php:579`
`  plugins/openvas/front/task.form.php:40             PluginOpenvasOmp::addTask()`
`2017-02-23 17:07:47 [19@xxx]`
`  *** PHP Notice(8): Undefined index: target`
`  Backtrace :`
`  plugins/openvas/inc/omp.class.php:580`
`  plugins/openvas/front/task.form.php:40             PluginOpenvasOmp::addTask()`
`2017-02-23 17:07:47 [19@xxx]`
`  *** PHP Notice(8): Trying to get property of non-object`
`  Backtrace :`
`  plugins/openvas/inc/omp.class.php:588`
`  plugins/openvas/front/task.form.php:40             PluginOpenvasOmp::addTask()`
`2017-02-23 17:07:47 [19@xxx]`
`  *** PHP Warning(2): Invalid argument supplied for foreach()`
`  Backtrace :`
`  plugins/openvas/inc/task.class.php:71`
`  plugins/openvas/front/task.php:47                  PluginOpenvasTask::showTasks()`

=> in sql-errors.log I see :
`  *** MySQL query error:`
`  SQL: SHOW COLUMNS FROM glpi_plugin_openvas_tasks`
`  Error: Table 'glpi.glpi_plugin_openvas_tasks' doesn't exist`
Effectively, this table does not exist. And I don't see in code where it can be created (nothing in tasks.class.php)

Am I missing something ?

Thx for help.

**Comment by aldiop** [comment]
Hi
Do you have some information regarding how to use this plugin

**Comment by keyser75000** [comment]
Hi,
Sorry for my late answer.
In fact the plugin works perfectly. I just forgot to install client side on my GLPI (OpenVas is another computer) : openvas-cli & librairies.
Tested on GLPI 9.1.3 / plugin 1.0.1 / OpenVAS-8

Thx for your work

**Comment by wawax** [comment]
Hi that's great !
I'd be interesting in a feedback ^^

**Comment by keyser75000** [comment]
Well I don't know what kind of feedback, so I'll explain the purpose in my company.
I knew that security team was looking at OpenVAS. I saw the glpi plugin so I decided to install it, as a "POC" (actually) then I made a demo.

Actually it's not on GLPI-Prod but in Qualif.
**Apps**
GLPI : 9.1.3 / fusioninventory / fields / news / reports / mreporting / webservices / monitoring / openvas
Shinken (with monitoring plugin) : 2.4.3
OpenVAS : v8

**Archi**
GLPI Server : Debian Wheezy 7.11 / OpenVAS libs 8.0.9
_Server will be upgraded this summer => Jessie_
OpenVAS Server : Debian Jessie 8.8 / OpenVAS libs 8.0.8 / Mnaager 6.0.9 / Scanner 5.0.7

Install was from sources. No packages available in v8 for Wheezy/Jessie. Use of sid/Ubuntu repository was not satisfying.
Others distribs are not qualified in ours environments (CentOS, Kali, Ubuntu), that's why I use Debian & Sources.
I'll try Strech since I see that it will include package v8.

**To Do :**
=> Production :
- OpenVAS Server re-install in DMZ (***rsync...).
- Migration of GLPI Server => Jessie & install Libs+CLI v8
- Docs / workflows / etc.
- Develop reports with glpi plugins as required.

=> Tests
- Test v9 (sources) with this plugin & Debian Jessie.
- Test Debian 9 with packages repository (only v8).
